errors/errorHandler.js

```diff
@@ -0,0 +1,9 @@
+
+
+function errorHandler(err, req, res, next) {
```

Comment:
Looks good but will want an error handler for 500 errors, would reccomend splitting this up. Have a look at the examples near the end of the error handling notes.